doc/doxygen/headers/glossary.h

```diff
@@ -378,6 +380,28 @@
  * @see @ref boundary "The module on boundaries"
  *
  *
+ * <dt class="glossary">@anchor GlossColorization <b>Colorization</b></dt>
+ * <dd><em>Colorization</em> is the process of marking certain parts of a
+ * Triangulation with different labels. The use of the word <em>color</em>
+ * comes from cartography, where countries on a map are made visually distinct
+ * from each other by assigning them different colors. Using the same term
+ * <em>coloring</em> is common in mathematics, even though we assign integers
+ * and not hues to different regions. deal.II refers to two processes as
```

Comment:
I love this explanation! I never thought about it in these terms -- thank you for making that leap for me!